app/src/main/java/vn/edu/usth/minigh/AuthActivity.java

```diff
@@ -0,0 +1,14 @@
+package vn.edu.usth.minigh;
+
+import androidx.appcompat.app.AppCompatActivity;
+
+import android.os.Bundle;
+
+public class AuthActivity extends AppCompatActivity {
+
+    @Override
+    protected void onCreate(Bundle savedInstanceState) {
+        super.onCreate(savedInstanceState);
+        setContentView(R.layout.activity_auth);
+    }
```

Comment:
Not an actual suggestion, just want to say that `AppCompatActivity` actually defines quite some handy stuff.

```suggestion
    public AuthActivity() {
        super(R.layout.activity_auth);
    }
```

Reply:
The `Bundle savedInstanceState` helps with switching orientation, which I don't think that we have time to do (?)

Reply:
When you don't override `onCreate`, of course `super.onCreate` is called.  In order to load from save, you'll need to save it first though and I don't know how to do it either.  There's nothing to be saved on the login view though so it's safe to not think about it *right now*.